sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -820,19 +802,21 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       applications = await this.getApplicationsToSubmitReassessment(offeringId);
 
       for (const application of applications) {
-        application.currentAssessment = {
-          application: { id: application.id } as Application,
-          triggerType: AssessmentTriggerType.OfferingChange,
-          offering: { id: offeringId } as EducationProgramOffering,
-          creator: auditUser,
-          createdAt: currentDate,
-          submittedBy: auditUser,
-          submittedDate: currentDate,
-        } as StudentAssessment;
+        if (application.applicationStatus === ApplicationStatus.completed) {
```

Comment:
how about overwritten application?

Reply:
Overwritten applications will not be a part of this logic. 
i.e when we fetch applications to submit re-assessment, Cancelled and Overwritten applications are ignored there. 
![image](https://user-images.githubusercontent.com/54600590/186015204-d1d4e59e-6e74-462d-9e9e-8d00e1be0d83.png)

![image](https://user-images.githubusercontent.com/54600590/186015307-9154b987-4a1d-41fa-927d-9e41dadf522f.png)